thirdparty/php/zimbra-php/rpm/SPECS/php.spec

```
@@ -19,6 +19,8 @@ URL:                http://php.net
 The Zimbra PHP build
 
 %changelog
+* Mon 14 Feb 2022  Zimbra Packaging Services <packaging-devel@zimbra.com>  - VERSION-1zimbra8.7b3ZAPPEND
```

nit: remove a white space
```suggestion
* Mon 14 Feb 2022  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b3ZAPPEND
```

---

Thanks, this has been fixed